tests/subsys/fs/fat_fs_dual_drive/prj.conf

```diff
@@ -5,3 +5,5 @@ CONFIG_FS_FATFS_NUM_FILES=8
 CONFIG_FS_FATFS_NUM_DIRS=8
 CONFIG_DISK_DRIVER_RAM=y
 CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
Why do we need `CONFIG_MAIN_STACK_SIZE=2048` ? Just curious. Otherwise LGTM.

Reply:
Because if I didn't modify the STACK_SIZE, it would fail on qemu_x86 on CI. But it would pass on my local. So I tried to find the reason, and I found CONFIG_MAIN_STACK_SIZE=2048 on my local. And I try to change it to 512 on local, it would see the same failure on CI. So I think it should be STACK_SIZE too small that caused it failed on qemu_x86 on CI. So I magnify the STACK_SIZE to 2048 to fix this problem.

Reply:
Understood. I suppose ideally this should be a comment or in its own commit though.